src/main/java/edu/gatech/cs2340/trydent/GameObject.java

```diff
@@ -25,7 +25,7 @@
 
     private String name = "Untitled GameObject";
 
-    private Group fxNode;
+    protected Group fxNode;
```

Comment:
I made this protected so that children of GameObject could still modify the underlying javafx nodes. Do we want this to be the intended means of adding functionality?